### PR TITLE
Morse code implementation

### PR DESCRIPTION
**Pull request by GordonOtieno**
**Implemented Morse-code**
- Implemented gitflow.
- solved linters errors.
- Implemented decode methods.
- updated readME files

### CODE REVIEW

File: morse_code.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+end
+
+def process_code(chars)
+  @letters = chars.split
+  @current_word = ''
+  @letters.each do |letter|
+    converted_code(letter)
+    @current_word += converted_code(letter)
+  end
+  print "#{@current_word} "
```
Comment (TedLivist) [inline morse_code.rb]: - [ ] We don't need to print the `current_word` or add extra spacing. Rather we return it. I think you should change to this:

  ```
  "#{@current_word}"
  ```

File: morse_code.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+end
+
+def morse_decoder(string)
+  @words = string.split('  ')
+  @words.each do |word|
+    process_code(word)
+  end
+end
```
Comment (TedLivist) [inline morse_code.rb]: - [ ] You are merely looping over the sentence. You need to actually return the sentence. Here is an idea:

  ```
  def morse_decoder(string)
    @words = string.split('  ')
    @msg_arr = []
    @words.each do |word|
      @msg_arr << process_code(word)
    end
    @msg_arr.join(' ')
  end
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @GordonOtieno and @betses,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**